### PR TITLE
Include backreference list to dossier and documents api serialization.

### PR DESCRIPTION
**Pull request by phgross**
This PR adds an additional key to the dossier and documents api serialization which lists summaries of all backreferences. It uses plone.restapis default summary serializer for the backreferences.

This is used for display backreferences in the new UI, see https://4teamwork.atlassian.net/browse/CA-1970

## Checklist
- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

_Only applicable should be left and checked._

- API change:
  - [x] API Changelog entry (see [guide](https://4teamwork.atlassian.net/wiki/spaces/4TEAM/pages/451248812/API+Changelog+Guidelines))

### CODE REVIEW

File: opengever/api/serializer.py [hunk 1]
```diff
@@ -90,6 +94,28 @@ def extend_with_groupurl(result, context, request):
...
+    intids = getUtility(IIntIds)
+    catalog = getUtility(ICatalog)
+    relations = catalog.findRelations(
+        {'to_id': intids.getId(aq_inner(context)),
+         'from_attribute': reference_attribute_name})
+    ids = [relation.from_id for relation in relations if relation.from_object]
+
+    for intid in ids:
+        summaries.append(
+            getMultiAdapter((intids.queryObject(intid), request),
```
Comment (njohner) [inline opengever/api/serializer.py:112]: So we are actually fetching the objects here and then summary serializing them. I guess this is a small set of objects, so it's not really an issue, and we anyway do not have any other choice. 

As a sidenote, is the step over the `ids` actually necessary? couldn't we instead simply do
`summaries = [getMultiAdapter((relation.from_object, request),ISerializeToJsonSummary)() for relation in relations]`?

File: changes/CA-1970.feature [hunk 2]
```diff
@@ -1 +1 @@
-Include backreference list to dossier api serialization. [phgross]
+Include backreference list to dossier and documents api serialization. [phgross]
```
Comment (njohner) [inline changes/CA-1970.feature:1]: to -> in

**Review by njohner — APPROVED** [review]
Mainly the one comment, not sure why we need to go over the IntIds...
Otherwise LGTM

File: opengever/api/serializer.py (hunk 1 above)
Comment (phgross) [reply opengever/api/serializer.py:112]: 🙈 you’re totally right! thx

**Pull request merged**